src/osdc/ObjectCacher.cc

```diff
@@ -454,15 +468,18 @@ ObjectCacher::BufferHead *ObjectCacher::Object::map_write(ObjectExtent &ex,
         if (cur + max >= bh->end()) {
           // we want right bit (one splice)
           final = split(bh, cur);   // just split it, take right half.
+	  maybe_rebuild_buffer(bh);
```

Comment:
Nit: mismatched indentation in this function?

Reply:
No. new code use tab. old code uses all whitespace.

Reply:
@ukernel Can we keep it consistent?